packages/create-pantheon-decoupled-kit/src/templates/tailwindless-next/components/layout.module.css

```diff
@@ -0,0 +1,26 @@
+.layout {
+  display: flex;
+  flex-direction: column;
+  min-height: 100dvh;
+  min-width: 100dvw;
+}
+
+.layout > main {
+  margin-bottom: auto;
+}
+
+.footerCopy {
+  margin: var(--x-auto);
+}
+
+.footerCopy > [href] {
+  text-decoration: underline;
+}
+
+.footerCopy > [href$='nextjs.org/'] {
+  color: var(--wite);
+}
+
+.footerCopy > [href$='wordpress.com/'] {
+  color: var(--blue);
```

Comment:
Technically this seems to be using different blue than the WP branded blue used by the tailwind version.